### ISSUE TITLE
hindquarter is not a narrow synonym for sacral region (UBERON_0005473)

### ISSUE THREAD

**Issue by azankl** [obophenotype/uberon#1483]

**Comment by azankl** [comment]
hindquarter is listed as a narrow synonym for sacral region (UBERON_0005473). However, the hindquarter does not only include the sacral region but also the entire hind limb. See Merriam-Webster definition here: https://www.merriam-webster.com/dictionary/hindquarter
Hindquarter is closer to hind limb than to sacral region, though really it is the union of sacral region and hind limb.

**Comment by cmungall** [comment]
Good point. It seems hindquarter may also be defined differently in different places, e.g. sometimes including some ribs?

The question is, would it be useful to have the union (or more formally, mereological sum) concept distinct in uberon? Including all possible combinations can be problematic for usability as we end up with lots of partonomy paths.

If we do not include it as a distinct concept, is it worse to have it with a non-exact synonym to assist in search purposes?

**Comment by azankl** [comment]
my feeling is that hindquarter can be a butcher's term (and then includes some ribs), but in a living animal, it refers to the hindlegs. One could argue that hindquarters includes the sacral region, while hindlegs does not, but I think the most straightforward would be to make hindquarters a narrow synonym of hindlegs.
